src/ensembl/src/content/app/browser/browser-region-field/BrowserRegionField.test.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import faker from 'faker';
+
+import { BrowserRegionField } from './BrowserRegionField';
+import Input from 'src/shared/components/input/Input';
+import Tooltip from 'src/shared/components/tooltip/Tooltip';
+
+import { ChrLocation } from '../browserState';
+import { LoadingState } from 'src/shared/types/loading-state';
+import genomeKaryotypes from 'tests/data/browser/karyotypes';
```

Comment:
Perhaps use a function to generate random data that has the shape of a karyotype item?
The shape of these items will likely change; and what is a couple of lines change in the constructor function will otherwise be a lot of copy-pasting in the data file.
It's not like we need the exact values of the karyotype items here, is it?